### PR TITLE
Feature/personen detail page

### PR DESCRIPTION
**Pull request by krausweb**
- added styles and layout on the mobile and desktop version
- added new icons
- renamed the theme detail page
- fixed js lint notices

**Layout added with mock data, without back-end**

### CODE REVIEW

**Comment by krausweb** [comment]
@MBeckers please,
I need real data for 
1) Personen and 
2) Person detail page

**Comment by krausweb** [comment]
Everything is updated here.
The front-end layout updated according to the new real and mock data, from the back-end. 
Please need reapprove.

File: app/Http/Resources/Person.php [hunk 1]
```diff
@@ -26,10 +26,69 @@ public function toArray($request)
...
+            'lifeSource' => $this->life_source,
             'role' => $org ? $org->pivot->role : null,
             'function' => $org ? $org->pivot->role : null,
-            'photo' => 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_1280.png'
+            'photo' => 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_1280.png',
+            'email' => 'dielinke.koeln@stadt-koeln', // @todo - fix mock data
+            'phone' => '0221-221 27840', // @todo - fix mock data
+            'fax' => '0221-221 27840', // @todo - fix mock data
+            'location' => 'Bezirksrathaus Rodenkirchen, Hauptstr. 85, 50996 Köln', // @todo - fix mock data
+            'committeeList' => // @todo - fix mock data
```
Comment (MBeckers) [inline app/Http/Resources/Person.php]: will fix when back in germany

**Comment by krausweb** [comment]
It works, but here back-end data is empty, because it seems that it does not work

**Pull request merged**